### ISSUE TITLE
Update dependencies

### ISSUE THREAD

**Issue by kevgo** [pilwon/node-get-env#2]
When installing this library, I get the warning:

```
npm WARN deprecated lodash@2.4.2: lodash@<3.0.0 is no longer maintained. Upgrade to lodash@^4.0.0
```

Updating to lodash 4.x will fix this.

**Comment by pilwon** [comment]
@kevgo Published as v0.4.1

**Comment by kevgo** [comment]
Awesome, thanks for the fast turnaround!
